File: dota-zone/dotazone/src/main/java/br/com/dotazone/DotaZoneBrain.java
package br.com.dotazone;

import com.prof.rssparser.Article;

import java.util.ArrayList;
import java.util.List;

import br.com.dotazone.model.entity.Hero;
import br.com.dotazone.model.entity.Item;

public class DotaZoneBrain {

    public static final String TAG = DotaZoneBrain.class.getName();
    public static final String RATING_DOTA_ZONE = "rating_dota_zone";
    public static List<Article> rssItems;
    public static List<Item> items = new ArrayList<Item>();
    public static List<Hero> heroes = new ArrayList<Hero>();
    public static Hero hero;
    //TODO Issue https://github.com/douglasalipio/dotazone/issues/2
    public static boolean isPremium = true;

}

File: dota-zone/dotazone/src/main/java/br/com/dotazone/DotaZoneBrain.kt
package br.com.dotazone

import br.com.dotazone.model.entity.Hero
import br.com.dotazone.model.entity.Item
import com.prof.rssparser.Article
import java.util.*

object DotaZoneBrain {
	val TAG = DotaZoneBrain::class.java.name
	const val RATING_DOTA_ZONE = "rating_dota_zone"
	var rssItems: List<Article>? = null
	var items: MutableList<Item> = mutableListOf()
	var heroes: MutableList<Hero> = mutableListOf()
	var hero: Hero? = null

	//TODO Issue https://github.com/douglasalipio/dotazone/issues/2
	var isPremium = true
}

